skills/release-loop/scripts/run-artifact-integrity.py

```
@@ -551,6 +567,348 @@ def legacy_manifest_digest(entries: list[dict[str, object]]) -> str:
+        if prefix is None:
+            return None
+        alias = prefix[1][1:]
+        resolved = anchors.get(alias, alias if alias in PROTECTED_V1_KEYS else "")
+        if not resolved:
+            return None
+        anchor = next((token[1][1:] for token in prefixes if token[0] == "&"), None)
+        return resolved, False, anchor
+    scalar = _yaml_scalar(value, start, anchors)
+    if scalar is None:
+        return None
+    resolved, end = scalar
```

Ruff (0.16.3)

[warning] Unpacked variable `end` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)